engine/src/flutter/display_list/dl_color.h

```diff
@@ -190,7 +190,7 @@ struct DlColor {
   DlColorSpace color_space_;
 
   static constexpr DlScalar toF(uint8_t comp) { return comp * (1.0f / 255); }
-  static constexpr uint8_t toC(DlScalar fComp) { return round(fComp * 255); }
+  static inline uint8_t toC(DlScalar fComp) { return std::round(fComp * 255); }
```

Comment:
toC, but not toF?

Reply:
toF doesn't use `std::round` which is constexpr since 23..